src/main/java/formflow/library/data/UserFileRepository.java

```diff
@@ -1,17 +1,28 @@
 package formflow.library.data;
 
-import java.util.List;
-import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+import java.util.UUID;
+
 /**
  * Repository interface for UserFile objects
  */
 @Repository
 public interface UserFileRepository extends JpaRepository<UserFile, UUID> {
 
+  /**
+   * finds all the {@link UserFile}s associated with a {@link Submission}}
+   * @param submission the {@link Submission} for which the associated {@link UserFile}s are sought
+   * @return {@link List} of associated {@link UserFile} objects
+   */
   List<UserFile> findAllBySubmission(Submission submission);
 
+  /**
+   * get a count of all the {@link UserFile}s associated with a {@link Submission}}
```

Comment:
This should also be capitalized

Reply:
Also using 3rd person consistently. Changed to "Gets".